docs/it/testing.md

```diff
@@ -167,6 +167,27 @@ Feature: Google Searching
     Then results for "panda" are shown
 ```
 
+### Mutation Testing
+
+Il mutation testing è una metodologia di test per valutare l'attuale test suite. Partendo dal codice, si creano diverse versioni diverse, ognuna delle quali introduce una mutazione, piccola a piacere. Queste versioni mutate vengono chiamati _mutanti_. L'obiettivo del mutation testing è di verificare se la test suite identifica la mutazione: se un mutante non viene rilevato da un test, viene considerato _vivo_ e i test a riguardo vengono considerati non sufficienti per quella mutazione. Se un mutante viene rilevato da un test, viene considerato _morto_ e i test a riguardo vengono considerati sufficienti per quel mutante.
```

Comment:
Mi aggancio a @guizzo per dire che questa sezione forse é da rivedere, l'argomento é un filo complesso e forse tra ripetizioni e forma, i concetti non arrivano in modo efficace infatti ho dovuto rileggerla un paio di volte, andrebbe solo rivista la struttura della frase a mio avviso